docs/Lama2/docs/explanation/lsp.md

```diff
@@ -0,0 +1,26 @@
+### Life Cycle of events in l2-lsp
```

Comment:
Let's include this file in the documentation MR.

Reply:
Yes

Reply:
You can remove from this MR.